packages/connector-jsdom/src/evaluate-runner.ts

```diff
@@ -28,7 +40,19 @@ const run = async (data: { source: string }) => {
         debug(err);
     });
 
-    const jsdom = await JSDOM.fromURL(url, {
+    let html = '';
+
+    try {
+        const networkData = await requester.get(url);
+
+        html = networkData.response.body.content;
+    } catch (error) {
+        process.send!({ error });
+
+        return;
+    }
+
+    const jsdom = new JSDOM(html, {
         beforeParse: beforeParse(url),
         pretendToBeVisual: true,
         resources: 'usable',
```

Comment:
Shouldn't we use the resource loader you've created to make sure all the assets are downloaded? Otherwise CSS files and other scripts might not be downloaded. Not sure that will impact a lot the output of axe but just for being extra cautious.

Reply:
sure, we can do that.

Reply:
I add a custom resource loader, but that is can't be used for JSDOM.fromURL, because JSDOM.fromURL is expecting that the promise has a `response` property.